modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console to see how the traffic moves around the application, in the menu bar, click the *Application launcher* > *Kiali*.
```

Comment:
>To access the console to see how the traffic moves around the application

Rather than mention just one thing you can see in Kiali, I would simply say "To access the Kiali console, do X"